terraform.tf

```diff
@@ -1,3 +1,3 @@
 terraform {
-  required_version = "~> 0.11.11"
+  required_version = "~> 0.12.3"
```

Comment:
This is problematic as it makes it so it stops working for TF 0.11 deployments. I _think_ the answer might need to be to pull this out entirely or replace it with a tfvar (if that's even possible).